keyserver/coniksserver/cmd/run.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"path"
+
+	"github.com/coniks-sys/coniks-go/keyserver"
+	"github.com/spf13/cobra"
+)
+
+// runCmd represents the run command
+var runCmd = &cobra.Command{
+	Use:   "run",
+	Short: "Run a CONIKS server instance",
+	Long: `Run a CONIKS server instance
+
+This will look for config files with default names (config.toml) 
+in the current directory if not specified differently.
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+		dir := cmd.Flag("dir").Value.String()
+		config := cmd.Flag("config").Value.String()
+
+		confPath := path.Join(dir, config)
```

Comment:
I see you changed `config` to just a filename now.  Maybe leaving it as a path is ok, and checking for absolute here?